src/client/components/CommentFooter/Buttons.js

```diff
@@ -222,7 +222,7 @@ class Buttons extends React.Component {
         </span>
         {user.name && (
           <span>
-            <span className="CommentFooter__bullet" />
+            <span className="CommentFooter__bullet_br" />
```

Comment:
I think general convention is 2 `__` so this would be `CommentFooter__bullet__br`

Reply:
We are using BEM (http://getbem.com/introduction/).
It would be `CommentFooter__bullet__br` if that would be child of `CommentFooter__bullet`, but looks like it isn't.
It should be `CommentFooter__bullet--br` as modifier.